context.go

```diff
@@ -323,7 +328,7 @@ func (c *Context) Sqrt(d, x *Decimal) (Condition, error) {
 	tmp := new(Decimal)
 	// The algorithm in the paper says to use c.Precision + 2. 7 instead of 2
```

Comment:
Can you add a citation to the algorithm/paper being followed in the function header comment? Otherwise this comment is too cryptic to follow. You should probably do this for all papers followed